doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,46 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
-      sort_family ::= Set
-      | Prop
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      sort_family ::= Prop
       | SProp
+      | Set
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   A high-level tool for automatically generating
+   (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts.
+   Each :n:`@reference` is a different inductive type identifier belonging to
+   the same package of mutual inductive definitions.
+   The command generates the :n:`@ident`\s as mutually recursive
+   definitions. Each term :n:`@ident` proves a general principle of mutual
+   induction for objects in type :n:`@reference`.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the scheme name will be determined
+      automatically from the sorts involved.
+
+   :n:`Induction for @reference Sort @sort_family`
```

Comment:
```suggestion
   :n:`Induction`
```

I'd add another item that describes `@reference Sort @sort_family` rather than repeating 4 times.
Given that the descriptions for what remains is very short "dependent possibly recursive", "non-dependent possibly recursive", ..., they could formatted with bullet points (one line each, no extra space)

Reply:
How about hyperlinking to your example at line 1084 here?  The visible text could be just "See examples **here**".